python/feathub/feature_tables/sources/file_system_source.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         timestamp_field: Optional[str] = None,
         timestamp_format: str = "epoch",
         max_out_of_orderness: timedelta = timedelta(0),
+        is_bounded: bool = True,
```

Comment:
Is there any case where we should initialize a FileSystemSource with `is_bounded=False`?